<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>nodeConfigurationDialog</class>
 <widget class="QDialog" name="nodeConfigurationDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>384</width>
    <height>252</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Node Configuration</string>
  </property>
  <property name="windowIcon">
   <iconset resource="watcherResources.qrc">
    <normaloff>:/icons/resources/watcher.png</normaloff>:/icons/resources/watcher.png</iconset>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>210</y>
     <width>183</width>
     <height>34</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QToolButton" name="nodeColorButton">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>160</y>
     <width>88</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Node Color</string>
   </property>
  </widget>
  <widget class="QPushButton" name="labelFontButton">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>160</y>
     <width>89</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Label Font</string>
   </property>
  </widget>
  <widget class="QComboBox" name="labelComboBox">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>50</y>
     <width>241</width>
     <height>27</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>last octet</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>two octets</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>three octets</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>four octets</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>hostname</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>free form</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>none</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>55</y>
     <width>79</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Label  Text:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelWhichNode">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>20</y>
     <width>231</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QToolButton" name="labelColorButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>160</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Label Color</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>108</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Node Affected: </string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="nodeShapeLabel">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>80</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Node Shape:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QComboBox" name="nodeShapeComboBox">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>80</y>
     <width>241</width>
     <height>27</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Sphere (Circle)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Cube (Square)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Torus (Circle)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Teapot (Square)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Pyramid (Triangle)</string>
    </property>
   </item>
  </widget>
  <widget class="QSpinBox" name="nodeSizeSpinBox">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>120</y>
     <width>55</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="nodeShapeLabel_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>120</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Node Size:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="watcherResources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>nodeConfigurationDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>nodeConfigurationDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>labelColorButton</sender>
   <signal>clicked()</signal>
   <receiver>nodeConfigurationDialog</receiver>
   <slot>setNodeLabelColor()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>54</x>
     <y>102</y>
    </hint>
    <hint type="destinationlabel">
     <x>191</x>
     <y>82</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>labelFontButton</sender>
   <signal>clicked()</signal>
   <receiver>nodeConfigurationDialog</receiver>
   <slot>setNodeLabelFont()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>154</x>
     <y>103</y>
    </hint>
    <hint type="destinationlabel">
     <x>191</x>
     <y>82</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>nodeColorButton</sender>
   <signal>clicked()</signal>
   <receiver>nodeConfigurationDialog</receiver>
   <slot>setNodeColor()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>253</x>
     <y>102</y>
    </hint>
    <hint type="destinationlabel">
     <x>191</x>
     <y>82</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>labelComboBox</sender>
   <signal>activated(QString)</signal>
   <receiver>nodeConfigurationDialog</receiver>
   <slot>setNodeLabel(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>220</x>
     <y>63</y>
    </hint>
    <hint type="destinationlabel">
     <x>191</x>
     <y>82</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>nodeShapeComboBox</sender>
   <signal>activated(QString)</signal>
   <receiver>nodeConfigurationDialog</receiver>
   <slot>setNodeShape(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>220</x>
     <y>93</y>
    </hint>
    <hint type="destinationlabel">
     <x>191</x>
     <y>105</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>nodeSizeSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>nodeConfigurationDialog</receiver>
   <slot>setNodeSize(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>167</x>
     <y>133</y>
    </hint>
    <hint type="destinationlabel">
     <x>191</x>
     <y>125</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>setNodeColor()</slot>
  <slot>setNodeLabelFont()</slot>
  <slot>setNodeLabelColor()</slot>
  <slot>setNodeLabel(QString)</slot>
  <slot>setNodeShape(QString)</slot>
  <slot>setNodeProperty(QString)</slot>
  <slot>setNodeSize(int)</slot>
  <slot>nodeClicked(int)</slot>
 </slots>
</ui>
